src/audio_psp2.cpp

```diff
@@ -186,6 +187,7 @@ static int sfxThread(unsigned int args, void* arg){
 		free(sfx->audiobuf2);
 		sfx_decoder[sfx->id].reset();
 		free(sfx);
+		availThreads++;
 		
```

Comment:
That usage of a volatile variable has a race condition. It is not guaranteed that ++ or -- are one operation, could have a register load which could result in incorrect values.
Especially because volatile disables the optimizer. The purpose of volatile was never multithreading but for accessing memory mapped hardware IO ports.

Assuming the PSVita has this (should be in newlib, is just normal ARM assembly stuff):
remove the volatile, include <atomic> and change the variable type to.
std::atomic<std::uint8_t>

Rest can stay the same, operators are overloaded.